### PR TITLE
Commerce 11483

### PR DESCRIPTION
**Pull request by MahmoudAzzam**
Jira : https://issues.liferay.com/browse/COMMERCE-11483

### CODE REVIEW

**Comment by MahmoudAzzam** [comment]
ci:test:relevant

**Review by andrea-ale-sbarra — CHANGES_REQUESTED** [review]
1. the first commit has a wrong commit message, see history for the right pattern.
2. we need to have actual values, not just add the fields.
I will count this as 1 rejection.

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-catalog-impl/src/main/java/com/liferay/headless/commerce/delivery/catalog/internal/dto/v1_0/converter/SkuDTOConverter.java [hunk 1]
```diff
@@ -153,6 +159,51 @@ public Sku toDTO(DTOConverterContext dtoConverterContext) throws Exception {
 						return commercePriceConfiguration.
 							displayDiscountLevels();
 					});
+				setReplacementSku(
+					() -> {
+						if (replacementCPInstance != null) {
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-catalog-impl/src/main/java/com/liferay/headless/commerce/delivery/catalog/internal/dto/v1_0/converter/SkuDTOConverter.java:164]: reverse this, check if this is == null and return null immediately

**Review by andrea-ale-sbarra — CHANGES_REQUESTED** [review]

**Review by andrea-ale-sbarra — CHANGES_REQUESTED** [review]
please see my comment

**Comment by MahmoudAzzam** [comment]
ci:test:sf

**Comment by MahmoudAzzam** [comment]
ci:test:relevant

**Review by andrea-ale-sbarra — APPROVED** [review]
LGTM.

Thank you!

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
Resent here https://github.com/liferay-commerce/liferay-portal/pull/3850

**Pull request closed without merging**